src/Centreon/Infrastructure/Monitoring/Resource/DbReadResourceRepository.php

```diff
@@ -0,0 +1,570 @@
+    {
+        $this->accessGroups = $accessGroups;
+        return $this;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function findResources(ResourceFilter $filter): array
+    {
+        $resources = [];
+
+        if ($this->hasNotEnoughRightsToContinue()) {
+            return $resources;
+        }
+
+        $collector = new StatementCollector();
+
+        $this->sqlRequestTranslator->setConcordanceArray($this->resourceConcordances);
+
+        $request = "SELECT SQL_CALC_FOUND_ROWS DISTINCT
+            resources.resource_id,
+            resources.name,
+            resources.alias,
+            resources.address,
+            resources.id,
+            resources.internal_id,
+            resources.parent_id,
+            resources.parent_name,
+            r2.status AS `parent_status`,
+            r2.alias AS `parent_alias`,
+            r2.status_ordered AS `parent_status_ordered`,
+            severities.level AS `severity_level`,
+            resources.type,
+            resources.status,
+            resources.status_ordered,
+            resources.status_confirmed,
+            resources.in_downtime,
+            resources.acknowledged,
+            resources.passive_checks_enabled,
+            resources.active_checks_enabled,
+            resources.notifications_enabled,
+            resources.last_check,
+            resources.last_status_change,
+            resources.check_attempts,
+            resources.max_check_attempts,
+            resources.notes,
+            resources.notes_url,
+            resources.action_url,
+            resources.output,
+            resources.poller_id,
+            resources.has_graph,
+            instances.name AS `monitoring_server_name`,
+            resources.enabled,
+            resources.icon_id
+        FROM `:dbstg`.`resources`
+        LEFT JOIN `:dbstg`.`resources` r2
+            ON r2.id = resources.parent_id
+        LEFT JOIN `:dbstg`.`severities`
+            ON `severities`.severity_id = `resources`.severity_id
+        LEFT JOIN `:dbstg`.`resources_tags` AS rtags
+            ON `rtags`.resource_id = `resources`.resource_id
+        INNER JOIN `:dbstg`.`instances`
+            ON `instances`.instance_id = `resources`.poller_id";
+
+        /**
+         * Handle search values
+         */
+        $searchSubRequest = null;
+        $hasWhereCondition = false;
+
+        try {
+            $searchSubRequest .= $this->sqlRequestTranslator->translateSearchParameterToSql();
+        } catch (RequestParametersTranslatorException $ex) {
+            throw new RepositoryException($ex->getMessage(), 0, $ex);
+        }
+
+        if (!empty($searchSubRequest)) {
+            $request .= $searchSubRequest;
+            $hasWhereCondition = true;
+        }
+
+        $request .= $hasWhereCondition ? ' AND ' : ' WHERE ';
+
+        $request .= " resources.name NOT LIKE '\_Module\_%'
+            AND resources.parent_name NOT LIKE '\_Module\_BAM'
```

Comment:
to handle ba monitored by remote server
```suggestion
            AND resources.parent_name NOT LIKE '\_Module\_BAM%'
```